### DISCUSSION TITLE
Refinement of survey responses - communication

### DISCUSSION THREAD

**Discussion by bertfrees** [daisy/pipeline#705] [General]
We are currently in the process of analysing your responses to the [Pipeline usage survey](https://github.com/daisy/pipeline/discussions/682) held earlier this year. Some of the feedback requires some more elaboration or discussion. Below are some responses in the topic "communication":

---
- **Q**: In your role, how well informed do you feel about Pipeline news and developments?
  - The majority of the organisations indicated they would like more information
- **Q**: Would you like your organization to be connected in a network with the DAISY Pipeline Development Team and other Pipeline developers/users?
  - The majority of the organisations indicated they would like to be connected in such a network
- **Q**: Have you used the Pipeline Source Code Repository? Any suggestions to make it more useful?
  - Someone suggested we should have an official way to contact the maintainers
     > Maybe we could have an official way to contact the maintainers, i.e. a mailing list or a Github discussion. ATM I just write to Bert directly
---

I would like to know:

- With the new [Github discussions forum](https://github.com/daisy/pipeline/discussions/659), the regular technical calls, and the announcements of the [quarter-yearly releases](https://github.com/daisy/pipeline/discussions/684) (such as the [one for the upcoming October release](https://github.com/daisy/pipeline/discussions/691)), do you feel sufficiently informed and connected?
- Are the current options to [connect with the maintainer team and community](http://daisy.github.io/pipeline/Get-Help/) sufficient?
- Where do you suggest the contact information could be added too?
- Other suggestions for improving the communication?

**Comment by egli** [comment]
> do you feel sufficiently informed and connected?

I think it is much better now and yes, sufficient

> Are the current options to [connect with the maintainer team and community](http://daisy.github.io/pipeline/Get-Help/) sufficient?

Yes

> Where do you suggest the contact information could be added too?

The [readme](https://github.com/daisy/pipeline/blob/master/README.md)  of the Pipeline repo should probably contain that info

> Other suggestions for improving the communication?

For me the "communication channel du jour" is [matrix.org](https://matrix.org/). It is kind of like an open source version of Slack. But then these thing come and go, so I'm not sure if we should bother

**Comment by josteinaj** [comment]
> With the new https://github.com/daisy/pipeline/discussions/659, the regular technical calls, and the announcements of the https://github.com/daisy/pipeline/discussions/684 (such as the https://github.com/daisy/pipeline/discussions/691), do you feel sufficiently informed and connected?
> Are the current options to [connect with the maintainer team and community](http://daisy.github.io/pipeline/Get-Help/) sufficient?

Yes, I think it's great that you've set up this discussion forum. And I don't have any problems getting in touch with the maintainer team.
